Gemfile.lock

```diff
@@ -148,8 +163,8 @@ GEM
     babel-transpiler (0.7.0)
       babel-source (>= 4.0, < 6)
       execjs (~> 2.0)
-    base64 (0.1.1)
-    bigdecimal (3.1.6)
+    base64 (0.2.0)
```

Comment:
oops i missed this: the prod error [from before](https://sassafras.slack.com/archives/CBHEDMM8Q/p1707260673351269) has come back, i'm gonna try to revert this specific upgrade